src/Roslyn/Conventional.Roslyn.Analyzers/Conventional.Roslyn.Analyzers.csproj

```diff
@@ -5,8 +5,8 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="3.6.0" />
-      <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.6.0" />
+      <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="4.5.0" />
```

Comment:
@murlakatam should we go with `4.7.0` here, given that is the latest version and what we are using in the test project, or do you need `4.5.0` for a specific reason?

Reply:
@andrewabest good point, I was constraint by another dependency of mine that was strictly wired to 4.5.0, but that's my pain. I will workaround that. Pushed a commit with 4.7.0 change